addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -810,7 +810,7 @@
                 <a t-attf-href="/jobs?#{'all_countries=1' if is_remote else current_country_path}#{non_location_params}"
                     t-attf-class="dropdown-item d-flex align-items-center justify-content-between nav-link#{'' if office_id or is_remote else ' active'}">
                     All Offices
-                    <span t-attf-class="badge rounded-pill#{' bg-primary' if office_id or is_remote else ' bg-light text-primary'} ms-2" t-out="count_per_office.get('all', '0')"/>
+                    <span t-attf-class="badge rounded-pill text-bg-primary ms-2" t-out="count_per_office.get('all', '0')"/>
```

Comment:
Same question as in other places.. no need for differentiation?

Reply:
Answered [here](https://github.com/odoo/odoo/pull/162935#discussion_r1706970291)